### PR TITLE
PCRM-0000 - Add email functionality for LR04 and add email formatting

### PR DESCRIPTION
**Pull request by abbas-khan8**
IS3 - Add email feedback for LR04
IS11 - Fix email formatting

### CODE REVIEW

File: terraform/modules/lambda/lambda.tf [hunk 1]
```diff
@@ -52,6 +52,10 @@ module "LR-04" {
   cloudwatch_kms_key    = var.cloudwatch_kms_key
   s3_kms_key            = var.s3_kms_key
   log_retention_in_days = var.log_retention_in_days
+  email_ssm             = var.email_ssm_prefix #IS3
```
Comment (aparimoo-nhs) [inline terraform/modules/lambda/lambda.tf:55]: remove comment

File: test/unittests/lambdas/LR_04_feedback_failure/test_LR_04_feedback_failure.py [hunk 2]
```diff
@@ -25,30 +26,46 @@ def test_lr_04_handler_invalid_event_raises_key_error(
...
+        "\n   • Blocked Route Special District Marker - Must be 'B' or 'S'."
+        "\nInvalid Record on lines 18-19"
+        "\n   • GP Code - Must be a valid 7-digit numeric GMC National GP code and 1-6-digit alphanumeric Local GP code separated by a comma."
+        "\n   • Drug Dispensed Marker - Must be 'Y' or blank."
+        "\n   • Blocked Route Special District Marker - Must be 'B' or 'S'."
+        "\n\nPlease check and amend the file content and upload again.\n"
+    )
+
+    assert result["status"] == "success"
+    assert result["message"] == f"LR04 Lambda application stopped for jobId='{JOB_ID}'"
```
Comment (aparimoo-nhs) [inline test/unittests/lambdas/LR_04_feedback_failure/test_LR_04_feedback_failure.py:66]: This is a default message , are we not sending our own message ?

File: terraform/modules/lambda/LR-04/lambda.tf [hunk 3]
```diff
@@ -31,6 +31,9 @@ resource "aws_lambda_function" "LR-04-Lambda" {
   environment {
     variables = {
       AWS_S3_REGISTRATION_EXTRACT_BUCKET = var.source_bucket
+      LISTREC_EMAIL                      = var.listrec_email #IS3
```
Comment (aparimoo-nhs) [inline terraform/modules/lambda/LR-04/lambda.tf:34]: Remove comment

File: test/unittests/lambdas/LR_04_feedback_failure/test_LR_04_feedback_failure.py (hunk 2 above)
Comment (abbas-khan8) [reply test/unittests/lambdas/LR_04_feedback_failure/test_LR_04_feedback_failure.py:66]: It's customised the default message slightly, added the lambda name and the job id

**Review by HarveyNHS — APPROVED** [review]
Approved

**Pull request merged**